wait_spec.rb

```diff
@@ -124,15 +124,15 @@
       it "responds to Element methods" do
         decorator = browser.div.when_present
 
-        decorator.should respond_to(:exist?)
-        decorator.should respond_to(:present?)
-        decorator.should respond_to(:click)
+        expect(decorator).to respond_to(:exist?)
+        expect(decorator).to respond_to(:present?)
+        expect(decorator).to respond_to(:click)
       end
 
       it "delegates present? to element" do
         Object.class_eval do
           def present?
-            false
+            true
```

Comment:
Hmm, If we've defined it to be present to return false, can you help me understand why we would expect it to be true?